src/code.sh

```
@@ -203,13 +212,15 @@ _merge_with_sentieon_vcf() {
 _normalize_vcf() {
     # Normalize the final VCF using bcftools norm.
     # The normalized file overwrites final_vcf to keep usage consistent.
-    local temp_norm_vcf="${final_vcf%.vcf.gz}_normalized.vcf.gz"
+    if [[ "$opt_normalisation" == "true" ]]; then
```

Shellcheck (0.10.0)

[warning] opt_normalisation is referenced but not assigned.

(SC2154)